script/Arcueid/Day 5/05_00_ARC05_9.txt

```diff
@@ -245,12 +252,12 @@ I'm not worried about losing my nerve. Three minutes is simply the longest a hum
 [sha:6a4c22fecc3eb9396dfec177672f6b523ab4639b]{
 -- Page 573, Offset 10152.
 -- 「くそ―――くそ、くそ、くそ―――！
-"Shit――shit, shit, shit―――!
+"Shit――shit, shit, shit!
```

Comment:
is the dash warranted?

Reply:
Hmm, there is more of a pause after the first one than between the others. Slight though.